client/apps/Admin/src/components/BulkAssignmentForm.tsx

```
@@ -0,0 +1,263 @@
+                                <div className="bulk-assignment-form__error" role="alert">
+                                    {submitError}
+                                </div>
+                            )}
+
+                            {/* Users Multi-Select */}
+                            <div className="bulk-assignment-form__section">
+                                <label className="bulk-assignment-form__label">
+                                    בחר עובדים <span className="bulk-assignment-form__required">*</span>
+                                </label>
+                                <div className="bulk-assignment-form__multi-select">
+                                    {usersData?.users.map((user: { id: string; fullName: string }) => (
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)